lab-yunjoo/index.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+//imporing the greeting method from the greetings module
+const greet = require(__dirname + '/lib/greet').greet;
```

Comment:
I like that you name your module the same as the filename that it comes from!

it makes it easy to tell where code comes from :)